test/unit/elasticsearch_index_test.rb

```diff
@@ -63,20 +68,37 @@ def test_add_sends_updates_to_the_bulk_index_endpoint
         "title" => "TITLE ONE",
     })
 
-    # Note that this comes with a trailing newline, which elasticsearch needs
-    payload = <<-eos
-{"index":{"_type":"edition","_id":"/foo/bar"}}
-{"_type":"edition","link":"/foo/bar","title":"TITLE ONE","popularity":0.09090909090909091,"format":"edition"}
-    eos
+    payload = [
```

Comment:
`index.add` normally takes an array. The indexing methods try to support both arrays of doc hashes and elasticsearch-formatted strings like the one I removed here. The latter is only used by one caller (the bulk_load script) so it's misleading to have all our tests assume this format. (This code for this can be refactored but I don't want to do it in this PR)